identity/src/test/java/com/android/identity/securearea/SoftwareSecureAreaTest.java

```diff
@@ -578,13 +623,11 @@ public void testEcKeySigningAllCurves() {
     @Test
     public void testEcKeyEcdhAllCurves() {
         EphemeralStorageEngine storage = new EphemeralStorageEngine();
-        BouncyCastleSecureArea ks = new BouncyCastleSecureArea(storage);
+        SoftwareSecureArea ks = new SoftwareSecureArea(storage);
 
         // Because we're using curves that cannot be used for signatures we need to
-        // create an key used to sign the key attestations
-        ks.createKey("attestationKey",
-                new BouncyCastleSecureArea.CreateKeySettings.Builder()
-                        .build());
+        // set an attestation key since self-signed certificates won't work.
+        // TODO: do this
```

Comment:
Should we address this TODO in the commit msg/as a new issue?

Reply:
I removed this comment. It'll be apparent this is needed when resolving the TODO below about enabling tests for Edwards curves.